docs/src/installation_upgrade.md

```diff
@@ -236,6 +236,179 @@ When versions are not directly upgradable, the old version needs to be
 removed before installing the new one. This won't affect user data but
 only the operator itself.
 
+### Upgrading to 1.21 from a previous minor version
```

Comment:
See other comments. This top section should cover the new 1.19 and 1.20 patches,
or should be a subsection under a wider header